### PR TITLE
Fix php-fpm detection with Nix

### PR DESCRIPTION
**Pull request by shyim**
With Nix all binaries are under bin and not sbin

### CODE REVIEW

File: discovery.go [hunk 1]
```diff
@@ -199,6 +199,12 @@ func (s *PHPStore) discoverPHPViaPHP(dir, binName string) *Version {
 		PHPPath:     php,
 	}
 	fpm := filepath.Join(dir, "sbin", strings.Replace(binName, "php", "php-fpm", 1))
+	fpmBin := filepath.Join(dir, "bin", strings.Replace(binName, "php", "php-fpm", 1))
```
Comment (fabpot) [inline discovery.go:202]: Instead, can you check if the file exists in `sbin` and switch to `bin` if it does not?

Comment (shyim) [reply discovery.go]: changed :)

File: discovery.go [hunk 2]
```diff
@@ -198,7 +198,13 @@ func (s *PHPStore) discoverPHPViaPHP(dir, binName string) *Version {
 		FullVersion: v,
 		PHPPath:     php,
 	}
-	fpm := filepath.Join(dir, "sbin", strings.Replace(binName, "php", "php-fpm", 1))
+	fpm := filepath.Join(dir, "bin", strings.Replace(binName, "php", "php-fpm", 1))
+	fpmSBin := filepath.Join(dir, "sbin", strings.Replace(binName, "php", "php-fpm", 1))
```
Comment (fabpot) [inline discovery.go:202]: Can you inline this directly? No need to create a temp variable that might not be used.

Comment (shyim) [reply discovery.go]: like so? 😅

**Review by fabpot — APPROVED** [review]
Perfect

**Pull request merged**